LabManagement/PowerShell/BulkOperations/Examples/Send-Invitations.ps1

```diff
@@ -0,0 +1,40 @@
+[CmdletBinding()]
+param(
+    [parameter(Mandatory = $true, ValueFromPipelineByPropertyName = $true)]
+    [ValidateNotNullOrEmpty()]
+    [string] $CsvConfigFile,
+
+    [parameter(Mandatory = $true, ValueFromPipelineByPropertyName = $true)]
+    [ValidateNotNullOrEmpty()]
+    [string] $CsvOutputFile,
+
+    [parameter(Mandatory = $false, ValueFromPipelineByPropertyName = $true)]
+    [int] $ThrottleLimit = 10
+)
+
+Set-StrictMode -Version Latest
+$ErrorActionPreference = 'Stop'
+
+# Make sure the input file does exist
+if (-not (Test-Path -Path $CsvConfigFile)) {
+    Write-Error "Input CSV File must exist, please choose a valid file location..."
+}
+
+# Make sure the output file doesn't exist
+if (Test-Path -Path $CsvOutputFile) {
```

Comment:
Should we implement the same "force" pattern as the Add-LabUsers.ps1 script to overwrite the output file?

Reply:
Sure, I think you mean to get rid of this check to delete it...and simply force the overwrite instead?  Note that I just copied\pasted the other examples, so we may want to make the change across the board at a certain point.

Reply:
We have "Force" as a parameter, and the user can optionally have us overwrite the file.  You have this in your other script - can probably just copy the pattern